R_scripts/licorplot.R

```diff
@@ -61,11 +68,19 @@ ggE <- ggplot(err, aes(x = err$day, y = err$meanflux, color = Destination_Plot,
   theme(axis.text.x = element_text(angle = 90, hjust = 1))
 
 #----- Plot temperature vs. flux with regression line -----
-q10_plot <- ggplot(dat, aes(x = Temperature, y = Flux)) +
+q10_plot <- ggplot(dat, aes(x = T20, y = Flux)) +
   geom_point(data = dat, size = 1) +
   geom_line(data = dat, size = 1) +
   geom_smooth(method = "lm") +
   ggtitle("Temperature vs. Flux") +
-  labs(x = "Temperature (°C)", y = "Flux (µmol m-2 s-1)")
-print(q10plot)
-ggsave("../outputs/q10.pdf")
\ No newline at end of file
+  labs(x = "Temperature (?C)", y = "Flux (?mol m-2 s-1)")
+print(q10_plot)
+ggsave("../outputs/q10.pdf")
+
+#----- Plot collar vs. CV with regression line -----
+ggCV <- ggplot(data = cv, aes(x = Collar, y = CV, color = n)) +
+  geom_point(data = cv) +
+  #geom_text_repel(data = cv, aes(label = Collar)) +
+  geom_smooth(method = "lm")
```

Comment:
Probably not necessary. We don't have any expectation that CV will be linearly related to collar number.